### PR TITLE
Site Profiler: Show the values on basic metrics as seconds when too big

### PR DESCRIPTION
**Pull request by WBerredo**
## Proposed Changes
* Display values bigger than `1000ms` in seconds
* Replace `’` per `'` the correct single quote character
* Set a fixed height for text containers to keep the sections aligned

## Why are these changes being made?

Discussions here: p1718354334303069-slack-C06G283R4LF

## Testing Instructions

* Go to `/site-profiler` and look for a site. Ex: `atliq.com`
* Click on the `Check site` button
* Verify all the style changes matches the ones below

| Before  | After |
| ------------- | ------------- |
|CleanShot 2024-06-14 at 10 48 41@2x|CleanShot 2024-06-14 at 10 46 46@2x|

### CODE REVIEW

File: client/site-profiler/utils/format-ms-value.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { I18N } from 'i18n-calypso';
+
+export const formatMsValue = (
+	msValue: number,
+	translate: I18N[ 'translate' ]
+): React.ReactElement | string | number => {
```
Comment (gcsecsey) [inline client/site-profiler/utils/format-ms-value.ts]: Nit: Can't we import the `translate` function here and use it from the function? It seems a bit redundant to me to always pass it with each call.

```suggestion
import { translate } from 'i18n-calypso';

export const formatMsValue = (
	msValue: number,
): React.ReactElement | string | number => {
```

**Review by gcsecsey — APPROVED** [review]
LGTM and works great! 🚢

File: client/site-profiler/utils/format-ms-value.ts (hunk 1 above)
Comment (WBerredo) [reply client/site-profiler/utils/format-ms-value.ts]: Yep, this works really well, thanks for the suggestion

https://github.com/Automattic/wp-calypso/pull/91806/commits/232aca9b9213969f20704ae5d93f52ee89b9e336

**Review by gcsecsey — APPROVED** [review]
LGTM 👌

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/14626296

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Thank you @WBerredo for including a screenshot in the description! This is really helpful for our translators.

**Comment by epeicher** [comment]
@gcsecsey , @WBerredo,  just stopping by to say thanks for the change. This simple change significantly improves the user experience.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/14626296).

**Pull request merged**